lib/std/os/windows.zig

```diff
@@ -3206,6 +3206,15 @@ pub const FILE_FS_DEVICE_INFORMATION = extern struct {
     Characteristics: ULONG,
 };
 
+pub const FILE_FS_VOLUME_INFORMATION = extern struct {
+    VolumeCreationTime: LARGE_INTEGER,
+    VolumeSerialNumber: ULONG,
+    VolumeLabelLength: ULONG,
+    SupportsObjects: BOOLEAN,
+    // Flexible array member
+    // VolumeLabel: [1]WCHAR,
```

Comment:
This should be included in the struct definition.

Reply:
Ah, I just followed the way `FILE_NOTIFY_INFORMATION ` is defined. Is it not the same sort of thing?
https://github.com/ziglang/zig/blob/91b02c0c5c16a2aee484b76d65a9442f8b71036b/lib/std/os/windows.zig#L3923-L3929

Reply:
Most other struct definitions include the field. It's helpful to include it since it means you can get a pointer to the field and then use that to get the full slice.

Example:

https://github.com/ziglang/zig/blob/7aaebd17746dc97111585563c64e166e27b334af/lib/std/os/windows.zig#L4774

https://github.com/ziglang/zig/blob/7aaebd17746dc97111585563c64e166e27b334af/lib/std/fs/Dir.zig#L465

Reply:
Okay I'll include it in my next push. Now a noob question, should I go ahead and simplify my commit history while I'm at it?

- `windows.zig: make WaitForMultipleObjectsEx assert MAXIMUM_WAIT_OBJECT…`
- `Watch.zig: add initial windows implementation` (and include the addition of FILE_FS_VOLUME_INFORMATION in windows.zig because it was literally added just for this)

I just want to make sure I'm doing things correctly lol.

Reply:
My understanding is that squashing fixup commits is preferred, so squashing all the `Watch.zig:` commits into one commit seems like the way to go.